webpack/assets/javascripts/react_app/components/factCharts/FactChart.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
```

Comment:
can you change the directory name to `FactCharts`?

Reply:
yes